core/shared/platform/ego/shared_platform.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set (PLATFORM_SHARED_DIR ${CMAKE_CURRENT_LIST_DIR})
+
+add_definitions(-DBH_PLATFORM_EGO)
```

Comment:
How about `BH_PLATFORM_OPEN_ENCLAVE`

Reply:
he specification is from EGO not Open Enclave.